docs/installation.rst

```diff
@@ -0,0 +1,69 @@
+Basic installation
+==================
+
+The simplest way to install the Parcels code is to use Anaconda and the `Parcels conda-forge package <https://anaconda.org/conda-forge/parcels>`_ with the latest release of Parcels. This package will automatically install all the requirements for a fully functional installation of Parcels. This is the “batteries-included” solution probably suitable for most users. If you want to install the latest development version of Parcels and work with features that have not yet been officially released, you can follow the instructions for a `developer installation <#installation-for-developers>`_.
+
+The steps below are the installation instructions for Linux / macOS and for Windows. If the commands for Linux / macOS and Windows differ, this is indicated with a comment at the end of the line.
+
+#. Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/, making sure to select the Python-3 version. If you're on Linux /macOS, it also assumes that you installed Miniconda-3 to your home directory.
+
+#. Start a terminal (Linux / macOS) or the Anaconda prompt (Windows). Activate the root (or base) environment of your Miniconda and create an environment containing Parcels, all its essential dependencies, and the nice-to-have cartopy and jupyter packages:
+
+    .. code-block:: bash
+
+      conda activate root  # Linux / macOS
+      activate root        # Windows
+
+      conda create -n parcels -c conda-forge parcels cartopy
+
+    .. note::
+
+        For some of the examples, ``pytest`` needs to be installed with ``conda install -n parcels pytest``
```

Comment:
Clarification (1. I was unfamliar with `-n` in `conda install`, and 2. previously may have been misread as `conda install parcels pytest` which installs `parcels` and `pytest`).

```suggestion
        For some of the examples, ``pytest`` also needs to be installed. This can be quickly done with ``conda install -n parcels pytest`` which installs ``pytest`` directly into the newly created ``parcels`` environment.
```